.release-please-manifest.json

```diff
@@ -3,5 +3,6 @@
   "packages/grz-pydantic-models": "2.1.0",
   "packages/grz-db": "0.2.1",
   "packages/grz-common": "1.0.3",
-  "packages/grzctl": "0.2.4"
+  "packages/grzctl": "0.2.4",
+  "packages/grz-check": "0.1.0"
```

Comment:
I _think_ we don't add it manually here, `release-please` should add this after this PR merges?

Reply:
ah, good point